src/Reflectivity/MetaLink.class.st

```diff
@@ -263,26 +263,33 @@ MetaLink >> installOn: aNode [
 
 { #category : #'installing - link installer' }
 MetaLink >> installOnClassVarNamed: aClassVarName for: aClassOrObject option: option instanceSpecific: instanceSpecific [
```

Comment:
I think this API methods can also be simplified. It can be a user responsibility to choose a variable instance and pass it to install method.

Reply:
yes, we should see how to simplify things even further.